brvs-core/src/main/java/iroha/validation/transactions/core/storage/impl/mongo/MongoBlockStorage.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package iroha.validation.transactions.core.storage.impl.mongo;
+
+import static com.mongodb.client.model.Filters.eq;
+
+import com.mongodb.client.model.ReplaceOptions;
+import iroha.protocol.BlockOuterClass.Block;
+import iroha.validation.transactions.core.archetype.mongo.MongoBasedStorage;
+import iroha.validation.transactions.core.storage.BlockStorage;
+import iroha.validation.utils.ValidationUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class MongoBlockStorage extends MongoBasedStorage<MongoBlock> implements BlockStorage {
```

Comment:
This storage is not used right now. It will be used later?

Reply:
I was used before. Atm that's true it's not used